### PR TITLE
README: remove text about GPU mining

### PR DESCRIPTION
**Pull request by sandakersmann**
readme: Ethereum is Proof-of-Stake now

### CODE REVIEW

**Comment by fjl** [comment]
This edit is kind of weird, why not remove the whole section instead.

**Comment by sandakersmann** [comment]
Made a PR for that earlier, but people didn't like that.

**Comment by AgbaraMmuo1** [comment]
NEED TO GET SOMETHINGS CLEAR HERE!__

**Comment by fjl** [comment]
I see. The problem is, this section *is* still relevant in the 'private network' setting. However, the part about `ethminer` can probably be removed.

File: README.md [hunk 1]
```diff
@@ -321,7 +321,7 @@ also need to configure a miner to process transactions and create new blocks for
 
 #### Running a private miner
 
-Mining on the public Ethereum network is a complex task as it's only feasible using GPUs,
+Mining on the public Ethereum network was a complex task involving GPUs,
 requiring an OpenCL or CUDA enabled `ethminer` instance. For information on such a
 setup, please consult the [EtherMining subreddit](https://www.reddit.com/r/EtherMining/)
 and the [ethminer](https://github.com/ethereum-mining/ethminer) repository.
```
Comment (fjl) [inline README.md:327]:
```suggestion
```

Comment (fjl) [reply README.md:327]: Should also remove 'however' in the first sentence of next paragraph.

**Review by fjl — APPROVED** [review]

**Pull request merged**